packages/experimental-utils/package.json

```diff
@@ -1,54 +0,0 @@
-{
```

Comment:
Might make sense to split the removal of `experimental-utils` out into its own PR, and base this PR on top of that one?